src/docs/cli/deployments/create.md

```diff
@@ -0,0 +1,69 @@
+---
+title: create
+---
+
+# peridio deployments create
+
+Create a deployment.
+
+## Usage
+
+```
+peridio \
+  deployments \
+  create \
+  [OPTIONS] \
+  --api-key <api-key> \
+  --conditions <conditions> \
+  --firmware <firmware> \
+  --name <name> \
+  --organization-name <organization-name> \
+  --product_name <product_name>
+```
+
+## Flags
+
+`-h`, `--help`
+
+Prints help information
+
+`-V`, `--version`
+
+Prints version information
+
+## Options
+
+### Required
+
+`--api-key <api-key>`, `$PERIDIO_API_KEY`
+
+The API key used to authenticate and authorize the request.
+
+`--conditions <conditions>`
+
+A json formated structure where:
+
+- `tags` key is an array of strings with the device tags
+- `version` key is a version string that respects this format: https://hexdocs.pm/elixir/Version.html#module-requirements
+
```

Comment:
conditions feel weird, I'd appreciate any input ^^/

Reply:
Remove the `--conditions` option. Add a `--tags` option and a `--version` option. Passing tags is required, passing a version is not required.